src/style.css

```diff
@@ -1 +1,118 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
```

Comment:
Directly styling the `body` tag is not recommended. Please use classes for styling instead, except for `html` and `body`.